tests/unit/components/ilios-calendar-event-test.js

```diff
@@ -0,0 +1,17 @@
+import { moduleForComponent, test } from 'ember-qunit';
```

Comment:
This test belongs in the integration test which already exists for this component.